samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md

```diff
@@ -0,0 +1,127 @@
+# Flogo Application sample for Apache Pulsar Connector
```

Comment:
Please rename Heading as
Sample about Batching and Chunking feature in Apache Pulsar Connector.